console/network/src/canary_v0.rs

```diff
@@ -152,6 +152,11 @@ impl Network for CanaryV0 {
         snarkvm_parameters::canary::GenesisBytes::load_bytes()
     }
 
+    /// Returns the restrictions list as a JSON-compatible string.
+    fn restrictions_list_as_str() -> &'static str {
```

Comment:
I'm not sure about the naming here. I think I'd prefer `restrictions_list()` or `restrictions_list_json()` to be more clear about the intent.

Reply:
Appreciate the suggestion, but let's leave it as is for now.